docs/quickstart.md

```diff
@@ -0,0 +1,249 @@
+
+```bash
+make inspect
+```
+
+<figure markdown>
+![Post Deployment Script](assets/img/aclabs-quickstart-make-inspect.png "Post Deploy"){ width=800px }
+<figcaption>Lab Status Output (Click to Zoom)</figcaption>
+</figure>
+
+## Interacting with the Lab
+
+### SSH
+
+Once the lab is up and running, we can use the terminal in our Codespace to SSH into the nodes.
+
+??? tip "More real estate for those SSH sessions 🏠"
+    Increase the size of the terminal by selecting the `Maximize Panel Size` button to the right of the terminal:
+
+    <figure markdown>
+    ![Post Deployment Script](assets/img/aclabs-quickstart-maximize-panel.png "Post Deploy"){ width=800px }
+    <figcaption>Maximize the Terminal (Click to Zoom)</figcaption>
+    </figure>
+
+??? tip "Use tabs 📑"
+    Create a new tab for an SSH session by selecting `New Terminal` to the right of the terminal.
+
+    Alternatively, the following keyboard shortcut can be used to open a new tab on both Windows and macOS: ++ctrl+shift+single-quote++
+
+    Once opened, the new terminal tab can be renamed by right-clicking on the tab and selecting `Rename`
+
+    === "New Terminal"
+        <figure markdown>
+        ![Post Deployment Script](assets/img/aclabs-quickstart-new-tab.png "Post Deploy"){ width=800px }
+        <figcaption>New Terminal Tab (Click to Zoom)</figcaption>
+        </figure>
+
+    === "Rename the Tab"
+        <figure markdown>
+        ![Post Deployment Script](assets/img/aclabs-quickstart-rename-tab.png "Post Deploy"){ width=800px }
+        <figcaption>Rename the Terminal Tab (Click to Zoom)</figcaption>
+        </figure>
+
+    === "Party Time 🎉"
+        <figure markdown>
+        ![Post Deployment Script](assets/img/aclabs-quickstart-renamed-tabs.png "Post Deploy"){ width=800px }
+        <figcaption>Rename the Terminal Tab (Click to Zoom)</figcaption>
+        </figure>
+
+A list of `Lab Hosts` that are accessible via `SSH` from within the codespace can be viewed at any time from the terminal by entering the following command:
+
+```bash
+make inspect
+```
+
+To access a node, use the `ssh` command followed by `admin@hostname` as shown in the example below
+
+```bash
+ssh admin@A-SPINE1
+```
+
+??? question "What's the password? :lock:"
+    Unless stated otherwise, the default username of `admin` and password of `admin` is used for all nodes in an Arista Community lab
+
+### API
+
+All nodes are accessible via API from within the codespace. The codespace comes pre-packaged with tools like Python and the Arista [AVD](https://galaxy.ansible.com/ui/repo/published/arista/avd/), [CVP](https://galaxy.ansible.com/ui/repo/published/arista/cvp/), and [EOS](https://galaxy.ansible.com/ui/repo/published/arista/eos/) Ansible Galaxy collections.
+
+## Stopping the Lab
+
+When you're finished with the lab, simply close the codespace by exiting the browser window. By default, a GitHub Codespace is stopped after 45 minutes of idleness and deleted after 30 days of inactivity.
```

Comment:
Can we add something close to "How to be green and save money" banner from the other quickstart? I think it's important to communicate that default of 30 days is not good and a habit of deleting codespace manually is the best.

Reply:
All set. Title and content of the note has been updated.